frontend/test/src/container/Home.spec.js

```diff
@@ -18,29 +38,51 @@ describe('<Home />', () => {
         instance_name: 'instanceTest'
       }
     },
+    t: key => key,
     canCreateWorkspace: true,
-    renderAppPopupCreation: renderAppPopupCreationCallBack
+    renderAppPopupCreation: renderAppPopupCreationCallBack,
+    dispatch: dispatchMock,
+    registerCustomEventHandlerList: () => { },
+    registerLiveMessageHandlerList: () => { }
   }
 
-  const wrapper = shallow(
-    <HomeWithoutHOC {...props} t={key => key} />
-  )
+  const HomeWithHOC1 = withRouterMock(translateMock()(HomeWithoutHOC))
+  const HomeWithHOC2 = ({ sys }) => <Provider store={store}><HomeWithHOC1 {...props} system={sys} /></Provider>
+
+  const wrapper = mount(<HomeWithHOC2 {...props} sys={props.system} />)
+  const homeWrapper = wrapper.find(HomeWithoutHOC)
+  const homeInstance = homeWrapper.instance()
+
+  describe('TLM handlers', () => {
+    describe('eventType user', () => {
+      const tlmData = {
+        author: userFromApi,
+        user: userFromApi
+      }
+      describe('handleUserModified', () => {
+        homeInstance.handleUserModified(tlmData)
+        it('should call this.props.dispatch(updateUserUsername())', () => {
+          expect(updateUserUsernameCallBack.called).to.equal(true)
+        })
+      })
+    })
+  })
 
   describe('static design', () => {
     it('should render the root div', () =>
-      expect(wrapper.find('div.tracim__content').length).equal(1)
+      expect(homeWrapper.find('div.tracim__content').length).equal(1)
     )
 
     it('should not render if workspaceList is not loaded', () => {
-      wrapper.setProps({ system: { ...props.system, workspaceListLoaded: false } })
+      wrapper.setProps({ sys: { ...props.system, workspaceListLoaded: false } })
```

Comment:
I don't think this prop exists for component Home, so I don't think this test is doing what it's supposed to

Reply:
[It's not possible use setProps to a children when mounting a element](https://github.com/enzymejs/enzyme/issues/1925), so I based on this idea https://github.com/enzymejs/enzyme/issues/1925#issuecomment-490637648 and what I do is I create a prop sys in the component and I pass this prop to the children's prop `system`.
I can use the name `system` for this "fake" prop too if you find it more clear.

Reply:
mmh ok but, your component is in `wrapper` and it never do anything related to a prop named `sys` so I don't understand what you are testing here

Reply:
sys is passed to component HomeWithHOC2 which passes the sys value to the Home component's prop system.
I just update the test _should not render if workspaceList is not loaded_ to works with a `mount` element.

Edit: I changed the names in my last commit to be more understandable.